tests/test_optimizers.py

```diff
@@ -101,6 +102,49 @@ def test_unit_cell_gradient_descent_optimization(
     assert not torch.allclose(state.cell, initial_state.cell)
 
 
+def test_fire_optimization(
+    ar_supercell_sim_state: SimState, lj_model: torch.nn.Module
+) -> None:
```

Comment:
Another nice test would be to check that `frechet_cell_fire` and `unit_cell_fire` with fixed cells return the same result as `fire`.